integreat_cms/cms/templates/pages/_page_machine_translation_overlay.html

```diff
@@ -0,0 +1,101 @@
+                             class="hidden bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 mb-5">
+                            {% translate "None of the selected pages fullfills the required HIX score of " %}  {{ hix_threshold }}.
+                        </div>
+                        <ul id="machine-translation-overlay-pages">
+                        </ul>
+                        <ul id="machine-translation-overlay-pages-optional" class="hidden">
+                        </ul>
+                        <a id="machine-translation-overlay-expansion-trigger"
+                           data-default-text="{% translate "Show all" %}"
+                           data-alternative-text="{% translate "Show less" %}"
+                           class="text-blue-500 hidden">{% translate "Show all" %}</a>
+                    </div>
+                </div>
+                <div class="mt-4">
+                    <h3>
+                        <b>{% translate "Budget" %}</b>
+                    </h3>
+                    <table>
+                        <tr>
+                            <td id="machine-translation-overlay-current-budget-label">{% translate "Current budget" %}</td>
+                            <td class="pl-2" id="machine-translation-overlay-current-budget-result">{{ deepl_budget }}</td>
+                        </tr>
+                        <tr>
+                            <td id="machine-translation-overlay-budget-usage-label">{% translate "Number of words you would use" %}</td>
+                            <td class="pl-2" id="machine-translation-overlay-budget-usage-result"></td>
+                        </tr>
+                        <tr class="border-solid border-t-2">
+                            <td id="machine-translation-overlay-budget-remains-label">{% translate "Remaining budget: " %}</td>
+                            <td class="pl-2" id="machine-translation-overlay-budget-remains-result"></td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="mt-4 hidden" id="machine-translation-overlay-warning">
+                    <h3>
+                        <b>{% translate "Pages that can't be translated" %}</b>
+                    </h3>
+                    <div class="my-2 bg-yellow-100 border-l-4 border-yellow-500 text-yellow-600 px-4 py-3 mb-5"
+                         role="alert">
+                        <i class="mr-2" icon-name="alert-triangle"></i>
+                        {% translate "There are currently" %} <span id="machine-translation-overlay-warning-number"></span> {% translate "pages that can't be translated. These pages can't be translated because the text understandibility is too low." %} {% translate "Please improve it to a value of at least 15 to translate these pages. You can identify the pages, that couldn't be translated in the page tree via the translation status." %}
```

Comment:
Also, I've explained quite often that it's bad practise to split up translatable sentences into multiple segments. There is no harm in using html markup inside translations, maybe shorten the id name a bit to make it better readable in the translation file.